src/organizations/components/OrgProfileTab/index.tsx

```diff
@@ -81,6 +81,7 @@ const OrgProfileTab: FC = () => {
   const allowSelfRemoval = users.length > 1
   const showLeaveOrgButton = isFlagEnabled('createDeleteOrgs')
   const hasFetchedOrgDetails = orgDetailsStatus === RemoteDataState.Done
+  const hasFetchedStorageType = Boolean(org?.defaultStorageType)
 
```

Comment:
I though about using the `isOrgIOX` selector here and having the storage type set to something like: `isOrgIox ? 'IOx' : 'TSM'`

Decided against it because the boolean selector is lossy (we're losing data on what the endpoint actually sends), and trying to convert it back into a string will create headaches if we have the UI glossing over what the endpoint sends, especially if we change the name of the storage type or add more in the future.